specification/subscription/resource-manager/Microsoft.Subscription/preview/2020-01-01-preview/subscriptions.json

```diff
@@ -0,0 +1,431 @@
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/SupportPlan"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SupportPlan": {
+      "description": "A SupportPlan resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SupportPlanProperties",
+          "title": "SupportPlan properties"
+        }
+      }
+    },
+    "SupportPlanProperties": {
+      "description": "The properties of the support plan.",
+      "properties": {
+        "planType": {
+          "description": "SupportPlan Type",
+          "type": "object",
+          "$ref": "#/definitions/planTypeEnum"
+        },
+        "endDate": {
+          "type": "string",
+          "description": "the end date"
+        }
+      }
+    },
+    "ProxyResource": {
+      "description": "The Resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "SupportPlanName": {
+      "type": "object",
+      "description": "The supportPlan Name.",
+      "properties": {
+        "planType": {
+          "description": "SupportPlan Type",
+          "type": "object",
+          "$ref": "#/definitions/planTypeEnum"
+        }
+      }
+    },
+    "PurchaseSupportResponseResult": {
+      "description": "A list of containing support plan result.",
+      "properties": {
+        "planType": {
+          "description": "SupportPlan Type",
+          "type": "object",
+          "$ref": "#/definitions/planTypeEnum"
+        },
+        "endDate": {
+          "type": "string",
+          "description": "the end date"
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Result of the request to list operations. It contains a list of operations and a URL link to get the next set of results.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "List of operations."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results if there are any."
+        }
+      }
+    },
+    "Operation": {
```

Comment:
>Operation" [](start = 5, length = 10)

unused. pls remove (  since operationListResult that references this is also unused)

Reply:
done.